connector-runtime/connector-runtime-core/src/main/java/io/camunda/connector/runtime/core/inbound/DefaultInboundConnectorContextFactory.java

```diff
@@ -50,7 +50,7 @@ public DefaultInboundConnectorContextFactory(
 
   @Override
   public <T extends InboundConnectorExecutable<?>> InboundConnectorContext createContext(
-      final InboundConnectorDefinitionImpl newConnector,
+      final InboundConnectorData connectorData,
```

Comment:
Dont we use connector Data also in our Webhook? Mabye `InboundConnectorDetails` or something along the line would be more clearer?